app/styles/_search.scss

```diff
@@ -91,8 +91,8 @@
   }
 
   .list.list-activity {
-    margin-left: 0rem;
-    margin-right: 0rem;
+    margin-left: 0.4rem;
```

Comment:
`margin: 0 0.4rem` should work. No seperate left and right need to specify.

Reply:
sure, just a sec, will change that.